Web/Scotty/Internal/Types.hs

```diff
@@ -56,14 +65,14 @@ data Options = Options { verbose :: Int -- ^ 0 = silent, 1(def) = startup banner
                                               -- servers using `setFdCacheDuration`.
                        }
 
-instance Default Options where
-    def = Options 1 defaultSettings
```

Comment:
IMHO the use of the `Default` class isn't that bad for settings-like datatypes.

How about keeping the instances around for the next major release along with `defaultOptions` in order to reduce breaking changes?

Reply:
good call! I've restored the instances

Reply:
Thanks!